data-reporting-service/person-service/src/test/resources/rawDataFiles/PatientSchema.json

```diff
@@ -0,0 +1,899 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Patient Full",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "personUid": {
+      "oneOf": [
+        {
+          "type": "null",
+          "title": "Not included"
+        },
+        {
+          "type": "integer"
+        }
+      ]
+    },
+    "personParentUid": {
+      "oneOf": [
+        {
+          "type": "null",
+          "title": "Not included"
+        },
+        {
+          "type": "integer"
+        }
+      ]
+    },
+    "description": {
+      "oneOf": [
+        {
+          "type": "null",
+          "title": "Not included"
+        },
+        {
+          "type": "string"
+        }
+      ]
+    },
+    "addTime": {
```

Comment:
For fields like addTime, shouldn't the type be a datetime type? The current setup has most of the fields set up as strings, we can work out the data type of some of the sink types to avoid compatibility issues.